packages/amplify-graphql-api-construct-tests/src/__tests__/backends/migration/base/app.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import { App, Stack, Duration } from 'aws-cdk-lib';
+import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/graphql-api-construct';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJson = require('../package.json');
+
+const app = new App();
+const stack = new Stack(app, packageJson.name.replace(/_/g, '-'), {
+  env: { region: process.env.CLI_REGION || 'us-west-2' },
+});
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const importedAmplifyDynamoDBTableMap: Record<string, string> = require('../table-map.json');
```

Comment:
Could we extend the `StackConfig` structure used by `packages/amplify-graphql-api-construct-tests/src/__tests__/backends/configurable-stack/app.ts` with this map instead of continuing to propagate new backend configs? We shouldn't make heroic efforts to do so, but it'd be nice to reduce the number of custom-built stacks we've got floating around our E2E tests

Reply:
I didn't realize that we had this. I'll update to use that.